sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/SessionHandler.java

```diff
@@ -48,8 +42,6 @@ public SessionHandler(String connectionId, String hostname, String sessionName,
         Duration openTimeout, AmqpMetricsProvider metricProvider) {
         super(connectionId, hostname);
         this.sessionName = sessionName;
-        this.openTimeout = openTimeout;
-        this.reactorDispatcher = reactorDispatcher;
```

Comment:
We'll keep these two Ctr arguments (openTimeout, reactorDispatcher) as removing those may lead to breaking the impl type.